src/main/java/org/jabref/gui/consistency/ConsistencySymbolsDialog.fxml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import javafx.scene.control.ButtonType?>
+<?import javafx.scene.control.DialogPane?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import org.jabref.gui.icon.JabRefIconView?>
+<DialogPane xmlns:fx="http://javafx.com/fxml/1" prefHeight="200.0" prefWidth="200.0"
+            xmlns="http://javafx.com/javafx/8.0.121" fx:controller="org.jabref.gui.consistency.ConsistencySymbolsDialog">
+    <content>
+        <VBox spacing="10.0">
+            <HBox spacing="10.0">
+                <JabRefIconView glyph="HYPHEN"/>
```

Comment:
Names should reflect meaning. So something like CONSISTENCY_FIELD_REQUIRED.
Adapt for the others too.